Makefile

```diff
@@ -63,7 +64,7 @@ XML_CATALOG_FILES=$(shell bin/find-file /usr/local/etc/xml/catalog)
 BUILD_CODE := intel-$(ARCH_CODE)-$(OSX_CODE)
 BUILD_DIR := build/$(BUILD_CODE)
 DESTDIR := $(PWD)/stage/git-$(BUILD_CODE)-$(VERSION)
-SUBMAKE := $(MAKE) C_INCLUDE_PATH="$(C_INCLUDE_PATH)" CPLUS_INCLUDE_PATH="$(CPLUS_INCLUDE_PATH)" LD_LIBRARY_PATH="$(LD_LIBRARY_PATH)" TARGET_FLAGS="$(TARGET_FLAGS)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" NO_GETTEXT=1 NO_DARWIN_PORTS=1 prefix=$(GIT_PREFIX) DESTDIR=$(DESTDIR)
+SUBMAKE := $(MAKE) C_INCLUDE_PATH="$(C_INCLUDE_PATH)" CPLUS_INCLUDE_PATH="$(CPLUS_INCLUDE_PATH)" LD_LIBRARY_PATH="$(LD_LIBRARY_PATH)" TARGET_FLAGS="$(TARGET_FLAGS)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" NO_GETTEXT=1 NO_DARWIN_PORTS=1 prefix=$(GIT_PREFIX) GIT_BUILT_FROM_COMMIT=$(GIT_BUILT_FROM_COMMIT) DESTDIR=$(DESTDIR)
```

Comment:
Wondering if we should have DQuotes around the new value.  (and maybe the $(GIT_PREFIX) too.)

Reply:
I'll try it and, assuming nothing weird happens (like the quotes showing up in `git version --build-options`), update this.

Reply:
Didn't see any issues - updated!